Content.Client/Access/UI/AccessLevelControl.xaml.cs

```diff
@@ -0,0 +1,55 @@
+using System.Linq;
+using Robust.Client.AutoGenerated;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.XAML;
+using Robust.Shared.Prototypes;
+using Content.Shared.Access;
+using Content.Shared.Access.Systems;
+
+namespace Content.Client.Access.UI;
+
+[GenerateTypedNameReferences]
+public sealed partial class AccessLevelControl : GridContainer
+{
+    [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+
+    public readonly Dictionary<string, Button> ButtonsList = new();
+
+    public AccessLevelControl(List<string> accessLevels, IPrototypeManager prototypeManager)
```

Comment:
update this list to the new prototype id API, it should be something like `List<ProtoId<AccessLevelPrototype>>`, (then update the tryindex)

Reply:
I tried to do this, but as far as I could tell, it would require rewriting the access system api. I can do it, but breaking api is bad, so I don't know if it's worth doing.

Reply:
Seems to have been updated since then. Can you convert them all now?